doc/kinsol/guide/source/Usage/index.rst

```diff
@@ -527,57 +527,6 @@ negative, so a test ``retval`` :math:`<0` will catch any error.
   +--------------------------------------------------------+----------------------------------+------------------------------+
 
 
-.. c:function:: int KINSetErrFile(void * kin_mem, FILE * errfp)
```

Comment:
Looks like there are still some instances of `KINSetErrFile` and `KINSetErrHandlerFn` elsewhere in this file